narwhal/primary/src/primary.rs

```diff
@@ -116,6 +116,9 @@ impl Primary {
             name.encode_base64()
         );
 
+        // Only enabled if failpoints feature flag is set
+        types::initialize_failpoints();
```

Comment:
I think more appropriate place is to initialize failpoints once in sui-node, rather then separately in primary and worker?